frontend/lib/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -341,18 +341,14 @@ const Multiselect: FC<Props> = props => {
                       overrides: {
                         Svg: {
                           style: {
-                            width: "10px",
-                            height: "10px",
+                            // The action icon should be around 0.625% of the parent font size.
+                            width: `0.625em`,
```

Comment:
nit: I think you should be able to use normal quotation marks `"` since you don't use a variable in here (no template string needed)

Reply:
Changed 👍